src/view.js

```diff
@@ -129,12 +131,26 @@ Marionette.View = Marionette.AbstractView.extend({
   addRegion: function(name, definition) {
     var regions = {};
     regions[name] = definition;
-    return this._buildRegions(regions)[name];
+    return this.addRegions(regions)[name];
   },
 
-  // Add multiple regions as a {name: definition, name2: def2} object literal
+  // Add multiple regions as a {name: definition, name2: def2} object literal or
+  // a function that evaluates to such literal
   addRegions: function(regions) {
+    return this._addRegions(regions, arguments);
```

Comment:
It's the same as in the region manager `addRegions` and what is written in the docs:

> If you supply a function to addRegions, it will be
called with the RegionManager instance context and
*all the arguments* passed to addRegions.

https://github.com/marionettejs/backbone.marionette/blob/8b32480adb2e432352ea1f29224f125be677f6d7/src/region-manager.js#L22